domains/rahulps.json

```diff
@@ -5,6 +5,6 @@
       },
 
       "record": {
-        "URL": "https://myfolio-rahulps.netlify.app"
+        "CNAME": "myfolio-rahulps.netlify.app"
```

Comment:
Sorry, we don't support Netlify. You can only use a URL.

Reply:
Thanks I will host it somewhere else and create a new pr